pandas/core/series.py

```diff
@@ -4319,6 +4320,54 @@ def between(self, left, right, inclusive=True) -> "Series":
 
         return lmask & rmask
 
+    # ----------------------------------------------------------------------
+    # Convert to types that support pd.NA
+
+    def _convert_dtypes(
+        self: ABCSeries,
+        infer_objects: bool = True,
+        convert_string: bool = True,
+        convert_integer: bool = True,
+        convert_boolean: bool = True,
+    ) -> ABCSeries:
+        """
```

Comment:
I wouldn't repeat this doc-string here.

you can return the type "Series", no?

Reply:
> I wouldn't repeat this doc-string here.
> 
I removed that

> you can return the type "Series", no?

Cant' return `Series` because this is a method inside of `Series`, so flake8 complains.  Left it as `ABCSeries`

Reply:
no, use '"Series"' (Series) in quotes.

Reply:
done